.github/workflows/test_rocprim.yml

```diff
@@ -0,0 +1,92 @@
+# Copyright 2025 Advanced Micro Devices, Inc.
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```

Comment:
Match project/organization style and drop this copyright notice. The license is also MIT now and not Apache, as of https://github.com/ROCm/TheRock/commit/b526b6d042dd8bfd8682e311d1e03beb1bbb335f.